api/v1alpha1/integrationtestscenario_types.go

```diff
@@ -55,9 +55,9 @@ type PipelineParameter struct {
 
 // TestEnvironment contains the name and values of a Test environment
 type TestEnvironment struct {
-	Name          string                                           `json:"name"`
-	Type          applicationapiv1alpha1.EnvironmentType           `json:"type"`
-	Configuration *applicationapiv1alpha1.EnvironmentConfiguration `json:"configuration,omitempty"`
+	Name          string                                      `json:"name"`
+	Type          helpers.DeprecatedEnvironmentType           `json:"type"`
```

Comment:
Can the `DeprecatedEnvironmentType` be defined as part of this file instead of `helpers`?

Reply:
I think that moving it would resolve at least a part of the current CI failures.

Reply:
Judging by the latest CI failures, I think that all other structs from helpers should go here in order to be deepCopy-able along with the rest of the API.

Reply:
Yup I think that's a fair way to handle it.